src/components/bcros/businessDetails/Links.vue

```diff
@@ -172,7 +172,7 @@ const downloadBusinessSummary = async (): Promise<void> => {
     </span>
 
     <div class="mb-2">
-      <BcrosBusinessDetailsLinkActions v-if="!isDisableNonBenCorps() && currentBusiness.identifier" />
+      <BcrosBusinessDetailsLinkActions v-if="!!currentBusinessIdentifier && !isDisableNonBenCorps()" />
```

Comment:
changes in here are for when current business is undefined due to it being a business bootstrap